### PR TITLE
[17.0][MIG] hr_announcement: Migration to version 17.0

### PR DESCRIPTION
**Pull request by juanjosesegui-tecnativa**
@Tecnativa TT50061

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Add the temporary dependency with https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

File: test-requirements.txt
Comment (chienandalu) [inline test-requirements.txt:1]: This change should go in a separate commit so you can drop it easily

**Review by chienandalu — COMMENTED** [review]
Test and code review :+1: 

Only :point_down:

File: test-requirements.txt
Comment (juanjosesegui-tecnativa) [reply test-requirements.txt:1]: Is this correct with the new commit?

**Comment by chienandalu** [comment]
Yes :) now let's wait for the main one to be merged :)

**Comment by victoralmau** [comment]
Can you rebase to review it (`announcement` is already merged https://github.com/OCA/server-ux/pull/935)?

**Comment by juanjosesegui-tecnativa** [comment]
> Can you rebase to review it (`announcement` is already merged [OCA/server-ux#935](https://github.com/OCA/server-ux/pull/935))?

Hi @victoralmau, it's done. Did I do it correctly?

**Review by chienandalu — APPROVED** [review]
Tested :+1:

**Review by victoralmau — APPROVED** [review]

**Pull request merged**
